src/tests/system/tests/test_authentication.py

```diff
@@ -137,3 +137,58 @@ def test_authentication__using_the_users_email_address(client: Client, ad: AD, m
     assert client.auth.parametrize(method).password(
         "uSEr_3@alias-dOMain.com", "Secret123"
     ), "User uSEr_3@alias-dOMain.com failed login!"
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyAD)
+@pytest.mark.ticket(jira="RHEL-65848", gh=7690)
+@pytest.mark.parametrize("method", ["su", "ssh"])
+@pytest.mark.importance("high")
+def test_authentication__by_updating_id_provider_to_ldap(client: Client, provider: GenericADProvider, method: str):
+    """
+    :title: Login by updating id_provider to ldap
```

Comment:
Login to AD when id_provider is set to ldap

if the test is about the config not changing the config.